aggregator/handler/router/traffic_control_options.go

```diff
@@ -18,48 +18,28 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package common
+package router
```

Comment:
Looking at the changes, it appears it would simplify things a lot if all traffic control related files are in its own package under `handler/router/trafficcontrol/`, then you can have `options.go`, `config.go`, `router.go`, `controller.go` etc and remove `TrafficControl` from the names of traffic-control related types and methods.

Thoughts?

Reply:
sure, sg